templates/zerver/help/mute-a-user.md

```diff
@@ -0,0 +1,47 @@
+# Mute a user
+
+To stop receiving notifications from a certain user,
+you can mute them.
+
+Messages sent by muted users don't trigger notifications,
+don't contribute to unread counts on the left sidebar,
+and are hidden from the message view.
+
+Muted users don't appear in the right sidebar, and
```

Comment:
"don't appear in the right sidebar or the [private messages](...) list".

Reply:
Ok, done.